tests/pytorch_pfn_extras_tests/onnx/test_annotate.py

```diff
@@ -141,6 +155,7 @@ def _fn2(x):
     assert 'yyy' in node_attrs
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
This DeprecationWarning seems to be raised only on the first call of `export_testcase`.